loris/user_commands.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import shutil
+from configobj import ConfigObj
+
+
+CONFIG_FILE_NAME = 'loris2.conf'
+CONFIG_DIR_DEFAULT = '/etc/loris2'
+WSGI_FILE_NAME = 'loris2.wsgi'
+
+
+def _src_code_repo_root():
+    return os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+
+def _init_dir(d):
+    # Could do something here to warn if dir exists but permissions or
+    # ownership aren't sufficient.
+    if not os.path.exists(d):
+        os.makedirs(d)
```

Comment:
minor: you can reuse `mkdir_p` from `utils.py`, which does this and will handle the case where the directory pops into existence before you call `makedirs`.

Reply:
thanks @alexwlchan. Done.